tests/utils.py

```diff
@@ -108,4 +108,4 @@ def assertMultifileGeneratedOutput(self, cpp, py, errors=None, flags=None):
             converter.output(module, buf)
 
             # Compare the generated code to expectation.
-            self.assertEqual(adjust(content), buf.getvalue(), "Discrepancy in %s" % module)
+            self.assertEqual(adjust(content), buf.getvalue())
```

Comment:
I changed this because the output is better when using `py.test` to run the unit tests (and if the third argument is present, then the output is completely supressed).